README.md

```diff
@@ -90,6 +90,18 @@ Persisting five articles having each three different authors, each with differen
 $article = ArticleFactory::make(5)->with('Authors[3].Address.City.Country', ['name' => 'Kenya'])->persist();
 ```
 
+#### On the command line:
+Factories can also conveniently populate your database in order to test your application on the browser.
```

Comment:
Test your application on the browser ? Shouldn't it be in the command line ?
If not, what is the use case for making this available.
Also the title of the shouldn't the pull request title be 'fixture factories on the command line'. I'm not sure I get what is meant by 'inline' here.
Can you just explain a little more what a use case would be for this ?

Is it : 
* I'm testing a feature in my app with my browser but I would like to have more records to test this 
* So I go to the command line and I create a bunch of records quickly so I can test some feature more easily ?

Once this is clarified, I could go further with the review

Reply:
There are two main use cases.

The first one is how you described, inserting data in the DB in order to manually test the app. Example: the tester wants to test the pagination of the articles index page, the provided command will enable to persist 100 of articles, facilitating the testing. Another example: a customer complains that a given query is too slow for a particularly large amount fo data. The developer can easily insert a large set of data aiming at reproducing the artifact.

The second use case enables the developer, with the `--dry-run` option, to easily display the output of her factories.